### DISCUSSION TITLE
is the option "ipv4 and ipv6" enabled?

### DISCUSSION THREAD

**Discussion by gitrys** [qdm12/ddns-updater#768] [Q&A]
Hello everybody

Im not shure if this option "ipv4 and ipv6" is ebabled yet.
In the readme there is this option not available yet (https://github.com/qdm12/ddns-updater/blob/master/docs/strato.md) and the respective issue is still open (https://github.com/qdm12/ddns-updater/issues/507). But then the Releases page shows this:
"Configurable ipv4 and ipv6 query parameter keys" in the 2.6 Update.

So im not shure if this works or not, someone can help me here?

Best Regards

**Comment by qdm12** [comment]
> Im not shure if this option "ipv4 and ipv6" is ebabled yet.

No, you need to have 2 entries in your config.json to update both ipv4 and ipv6.


> In the readme there is this option not available yet (https://github.com/qdm12/ddns-updater/blob/master/docs/strato.md)

What option? I only see `ipv4 or ipv6` which is available, not `ipv4 and ipv6`.



> the respective issue is still open (#507)

You can subscribe to this issue to be alerted when this is done. It's not done for now.



> But then the Releases page shows this:
> "Configurable ipv4 and ipv6 query parameter keys" in the 2.6 Update.

This is for the **custom** provider only, check the parent bullet point. So you may be able to update both ipv4 and ipv6 if your custom update endpoint URL supports passing both ipv4 and ipv6 parameters.
